bundles/org.openhab.automation.jrubyscripting/src/main/resources/OH-INF/config/config.xml

```diff
@@ -102,5 +108,13 @@
 			<advanced>true</advanced>
 		</parameter>
 
+		<parameter name="console" type="text" required="false" groupName="console">
+			<label>Console Script</label>
+			<description><![CDATA[The script file to be required by the <tt>jrubyscripting console</tt>
+			console command.]]></description>
+			<default>openhab/console/irb</default>
```

Comment:
we can just default this to `irb` now, since the `jrubyscripting console` command will prefix it with "openhab/console/" if it doesn't contain a "/"

Reply:
good idea. done.